requirements.txt

```diff
@@ -1,8 +1,8 @@
 numpy~=1.23.5
-pandas
+pandas==2.0.3
```

Comment:
pandas was giving an error? I think we could probably remove pandas from here all together since I don't remember where I am using it.

Reply:
I checked all scripts and none of them contains a reference to pandas package. I'll remove it.

![image](https://github.com/user-attachments/assets/03c49685-7b80-459b-8aac-eb778696390d)